manifests/kube_addons.pp

```diff
@@ -24,15 +24,16 @@
     try_sleep   => 30,
   }
 
+  $exec_onlyif = ['kubectl get nodes']
```

Comment:
I don't think there is any benefit of this being an array. It can just be a string.